packages/components/src/components/input/input.lite.tsx

```diff
@@ -93,6 +98,9 @@ export default function DBInput(props: DBInputProps) {
 
 	onMount(() => {
 		state._id = props.id ? props.id : 'input-' + uuid();
+		state._dataListId = props.dataListId
+			? props.dataListId
+			: `datalist-${state._id}`;
```

Comment:
```suggestion
		state._dataListId = props.dataListId || `datalist-${state._id}`;
```

From my understanding this would be the equivalent.